commit 30643576e711f6b638cf50e6d30af6e29d8456d9
Author: Michael Cyr <mikecyr@us.ibm.com>
Date:   Fri Mar 17 00:48:22 2017 +0000

    scsi: ibmvscsis: Synchronize cmds at remove time
    
    [ Upstream commit 8bf11557d44d00562360d370de8aa70ba89aa0d5 ]
    
    This patch adds code to disconnect from the client, which will make sure
    any outstanding commands have been completed, before continuing on with
    the remove operation.
    
    Signed-off-by: Michael Cyr <mikecyr@us.ibm.com>
    Signed-off-by: Bryant G. Ly <bryantly@linux.vnet.ibm.com>
    Tested-by: Steven Royer <seroyer@linux.vnet.ibm.com>
    Signed-off-by: Martin K. Petersen <martin.petersen@oracle.com>
    Signed-off-by: Sasha Levin <alexander.levin@verizon.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1421_30643576e711f6b638cf50e6d30af6e29d8456d9.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1421_30643576e711f6b638cf50e6d30af6e29d8456d9.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
